src/platform-includes/capture-error/flutter.mdx

```diff
@@ -1,4 +1,4 @@
 <PlatformContent includePath="capture-error/dart.mdx" />
 
 - Flutter-specific errors, such as using `FlutterError.onError`, are captured automatically
-- The SDK already runs your init `callback` on an error handler, such as using `runZonedGuarded`, are captured automatically
+- Errors not cuaght by Flutter will be automatically recorded by the SDK using the [`PlatformDispatcher.onError`](https://api.flutter.dev/flutter/dart-ui/PlatformDispatcher/onError.html) callback. For Flutter versions lower than 3.3, the `AppRunner` you provide on init will be run inside a custom zone using [`runZonedGuarded`](https://api.flutter.dev/flutter/dart-async/runZonedGuarded.html) to records errors.
```

Comment:
I'd rather add a new item about the new onError, and both items mention the version constraint.

Reply:
The same has to be applied on https://docs.sentry.io/platforms/flutter/usage/#tips-for-catching-errors